dmscripts/export_framework_results_reasons.py

```diff
@@ -194,6 +161,14 @@ def create_row(self, record):
             contact_details(record)
 
 
+def get_questions_numbers_from_framework(framework_slug, content_loader):
```

Comment:
Another wtf here is that I didn't remove the original load_manifest call after moving it